### ISSUE TITLE
exported .mu files over twice the size of unity mus

### ISSUE THREAD

**Issue by Rodg88** [taniwha/io_object_mu#86]
Something I've noticed after editing imported .mu files and then exporting them, if the size of the .mu tends to at least double compared to the original .mu file.

Steps to reproduce - import a mu saved by parttools, and export it with a new name. the new .mu will be larger.
Here's a test very simple .mu (sphere with "Standard" shader, and one sphere collider), which is 24KB. after importing and exporting, it becomes 82KB.
[2mSphere.zip](https://github.com/user-attachments/files/16151176/2mSphere.zip)

Or some real part examples:
![image](https://github.com/taniwha/io_object_mu/assets/2611674/c7e064c6-bce2-4e42-a3b5-af425bfbe964)
![image](https://github.com/taniwha/io_object_mu/assets/2611674/4f409767-b653-4d9f-a141-8997a0a9c993)
(1206F actually has a simpler hierarchy/less objects than the plain 1206)

**Comment by popos123** [comment]
thats true, it happened with every part files, i was thinking that not a bug that a feature, but lets see what author will do.
